src/stdafx.h

```diff
@@ -28,6 +28,12 @@
 
 #if defined(__APPLE__)
 #	include "os/macosx/osx_stdafx.h"
+#else
+/* It seems that we need to include stdint.h before anything else
+ * We need INT64_MAX, which for most systems comes from stdint.h.
+ * For OSX the inclusion is already done in osx_stdafx.h. */
+#	define __STDC_LIMIT_MACROS
+#	include <stdint.h>
```

Comment:
Do we need to `#include <stdint.h>` at all now that we `#include <cstdint>`?

Reply:
I don't know. I don't want to pull that change into this PR. Feel free to make a separate PR to figure that out though.